www/js/control/ControlDataTable.jsx

```diff
@@ -6,11 +6,12 @@ import { array } from "prop-types";
 // Note the camelCase to dash-case conventions when translating to .html files!
 // val with explicit call toString() to resolve bool values not showing
 const ControlDataTable = ({ controlData }) => {
-  console.log("Printing data trying to tabulate", controlData);
+  // console.log("Printing data trying to tabulate", controlData);
   return (
+    //rows require unique keys!
```

Comment:
This is something I encountered as well (in https://github.com/e-mission/e-mission-phone/pull/993)
I see you've already figured it out; I think this will be the case anywhere we use `.map()` or otherwise have repeated instances of a component one after another

Reply:
wait, what is related to `map()`? I thought this would be because the values don't "stick" until the `$apply()` is called

Reply:
Ah I see that this comment was about the requiring unique keys